resources/blog/posts/2023/2023-08-21-using-scoped-traverse.md

```diff
@@ -0,0 +1,84 @@
+---
+id: 51
```

Comment:
This needs to be 52 if PR:

- https://github.com/rectorphp/getrector-com/pull/1589

merged first

Reply:
No worries, I'l use 52 on the test